notification_service/settings.py

```diff
@@ -186,13 +192,26 @@
 CORS_ORIGIN_ALLOW_ALL = env.bool("CORS_ORIGIN_ALLOW_ALL")
 
 AUTHENTICATION_BACKENDS = [
+    "helusers.tunnistamo_oidc.TunnistamoOIDCAuth",
     "axes.backends.AxesBackend",
     "django.contrib.auth.backends.ModelBackend",
 ]
 
 AUTH_USER_MODEL = "users.User"
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+# Tunnistamo settings
```

Comment:
Maybe mentioning that these are for Keycloak and the namings related to Tunnistamo are for historical reasons (or something else) could help make things clearer?

Reply:
Added.